spec/presenter_spec.rb

```diff
@@ -1,19 +1,23 @@
 describe "Presenter" do
+  let(:a) { Cell.alive }
+  let(:d) { Cell.dead }
+
   it "accurately outputs the grid" do
-    cells = {
-      0 => {
-        0 => Cell.new('alive'), 1 => Cell.new('alive'), 2 => Cell.new('dead')
-      },
-      1 => {
-        0 => Cell.new('alive'), 1 => Cell.new('alive'), 2 => Cell.new('dead')
-      },
-      2 => {
-        0 => Cell.new('alive'), 1 => Cell.new('alive'), 2 => Cell.new('dead')
-      }
-    }
+    cells = [
+      [a, a, d],
+      [a, d, a],
+      [d, d, a],
+    ]
 
     grid = Grid.new(3, 3, cells)
-    expect { grid.presenter.print }.to output("•• \n•• \n•• \n").to_stdout
+    s = Cell.alive.to_s
+    expected_print = [
+      "🐰 🐰  ",
+      "🐰   🐰",
+      "    🐰",
```

Comment:
Will this be able to be stored in a UTF8 database column? We've committed to supported MySQL 5.4 for another 6 months.